graspologic/plot/plot.py

```diff
@@ -547,9 +552,9 @@ def pairplot(
     X : array-like, shape (n_samples, n_features)
         Input data.
     labels : array-like or list, shape (n_samples), optional
-        Labels that correspond to each sample in ``X``.
+        Labels that correspond to each sample in X.
```

Comment:
also why this change

Reply:
Yea i did not make these changes... i am unsure what is going on here...